### ISSUE TITLE
Regression: Added vairables

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#200]
```
Keep the specified XY variables in the regression dialog after the residuals or 
predicted values are added to the table (currently the regression dialog is 
reset to empty, so all the variables in the model specification are gone).  
Also keep the "save to file" button is enabled (it's currently disabled after 
adding residuals or predicted values) -- otherwise users would lose their 
results after adding the variables if they hadn't saved them before. 
```

Original issue reported on code.google.com by `jkoschin...@gmail.com` on 16 Jul 2015 at 2:00

**Comment by lixun910** [comment]
fix is proposed in ci 3730 and 3731

fix will be applied in 1.7.47

**Comment by jkoschinsky** [comment]
Verified in GeoDa 1.7.63.

Verify font issue in Windows.

**Comment by jkoschinsky** [comment]
Verified fix in GeoDa 1.7.89 (Win 7 Professional (64-bit)).
